src/config.ts

```diff
@@ -251,7 +252,7 @@ export class Config extends EventEmitter {
    * secure links.
    */
   public getAESKey = () => {
```

Comment:
This is mostly for encoding file contents when saving to disk. I think a better approach is to _not_ have this key set when there's no token, as a restart and reload will cause it to fail since the key changes.

Reply:
decided to set an static key `browserless` when token is not present to avoid changes when saving to disk because a non empty key is required